### DISCUSSION TITLE
Weird problem with sidebar

### DISCUSSION THREAD

**Discussion by reliak** [facebook/docusaurus#8905] [General]
Hi,
since a few days, I have a problem when collapsing the sidebar. There is still text on the side and the expand-icon at the bottom is not shown correctly.
Another problem is, that the collapsing of sidebar items is not working correctly.
I have attached a gif, that shows the two problems. The gif was created when browsing on https://docusaurus.io/docs
![problem](https://user-images.githubusercontent.com/4062774/233126818-3beee73a-f6a1-41b2-8e96-18dbdf5a7443.gif)

I have these problems only with my computer (Windows 10). I tested multiple browsers and the problem occurs in all of them. When I test it on another computer, everything works fine, so it is not a bug.

Have anybody else experienced this problem or know a solution to this?
Any help is appreciated.

**Comment by slorber** [comment]
@reliak this looks related to https://github.com/facebook/docusaurus/discussions/8854

Have you configured Windows to display less animations? This is the reason.

Just double checked, and we have a bug in Docusaurus Collapsible component when animations are reduced.

Bug because of https://github.com/facebook/docusaurus/pull/8674

This can also be reproduced on Mac/Chrome on nested details elements:

<img width="988" alt="CleanShot 2023-04-19 at 18 10 56@2x" src="https://user-images.githubusercontent.com/749374/233136036-349d8753-2e3d-4c6e-bd3c-becf6a8a691e.png">

https://docusaurus.io/docs/markdown-features#details

I'm looking to fix it asap

**Reply by slorber** [reply]
https://github.com/facebook/docusaurus/pull/8906

**Reply by firasrg** [reply]
Thank God I found someone having same issue as me ! 🥹
